autopilot/autopilot.go

```diff
@@ -120,14 +146,13 @@ type Autopilot struct {
 }
 
 // New initializes an Autopilot.
-func New(cfg config.Autopilot, bus Bus, workers []Worker, logger *zap.Logger) (_ *Autopilot, err error) {
+func New(cfg config.Autopilot, mCfg config.Migrator, masterKey utils.MasterKey, bus Bus, logger *zap.Logger) (_ *Autopilot, err error) {
```

Comment:
I considered housing the migrator config under the autopilot config, which definitely makes sense but it somewhat conflicts with how our YAML config file is organised. I ended up making it its own separate section, to be consistent with our database configuration. I personally feel we should have a PR dedicated to reorganising and cleaning up the `renterd` config and merge this as is for now.